### PR TITLE
Add informations for MAMP

### PR DESCRIPTION
**Pull request by okom3pom**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add informations for MAMP
| Fixed ticket? | N/A

### CODE REVIEW

File: src/content/1.7/basics/installation/localhost.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Here is a selection of free AMP installers:
 * [XAMPP](https://www.apachefriends.org/en/xampp.html) (Windows, Mac OS X, Linux, Solaris)
 * [WampServer](http://www.wampserver.com/) (Windows)
 * [EasyPHP](https://www.easyphp.org/) (Windows)
-* [MAMP](https://www.mamp.info/) (Mac OS X)
+* [MAMP](https://www.mamp.info/) (Windows, Mac OS X)
```
Comment (matks) [inline src/content/1.7/basics/installation/localhost.md:26]:
```suggestion
* [MAMP](https://www.mamp.info/en/mamp/mac/) (Windows, Mac OS X)
```

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_localhost.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Here is a selection of free AMP installers:
 * [XAMPP](https://www.apachefriends.org/en/xampp.html) (Windows, Mac OS X, Linux, Solaris)
 * [WampServer](http://www.wampserver.com/) (Windows)
 * [EasyPHP](https://www.easyphp.org/) (Windows)
-* [MAMP](https://www.mamp.info/) (Mac OS X)
+* [MAMP](https://www.mamp.info/) (Windows, Mac OS X)
```
Comment (matks) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_localhost.md:40]:
```suggestion
* [MAMP](https://www.mamp.info/en/mamp/mac/) (Windows, Mac OS X)
```

**Comment by matks** [comment]
Hi @okom3pom it seems link to https://www.mamp.info/ is now a 403 so I took the opportunity to update the link. Thank you for your contribution !

File: src/content/1.7/basics/installation/localhost.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ Here is a selection of free AMP installers:
 * [XAMPP](https://www.apachefriends.org/en/xampp.html) (Windows, Mac OS X, Linux, Solaris)
 * [WampServer](http://www.wampserver.com/) (Windows)
 * [EasyPHP](https://www.easyphp.org/) (Windows)
-* [MAMP](https://www.mamp.info/) (Mac OS X)
+* [MAMP](https://www.mamp.info/en/mamp/mac/) (Windows, Mac OS X)
```
Comment (matks) [inline src/content/1.7/basics/installation/localhost.md:26]:
```suggestion
* [MAMP](https://www.mamp.info/en/mamp/) (Windows, Mac OS X)
```

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_localhost.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ Here is a selection of free AMP installers:
 * [XAMPP](https://www.apachefriends.org/en/xampp.html) (Windows, Mac OS X, Linux, Solaris)
 * [WampServer](http://www.wampserver.com/) (Windows)
 * [EasyPHP](https://www.easyphp.org/) (Windows)
-* [MAMP](https://www.mamp.info/) (Mac OS X)
+* [MAMP](https://www.mamp.info/en/mamp/mac/) (Windows, Mac OS X)
```
Comment (matks) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_localhost.md:40]:
```suggestion
* [MAMP](https://www.mamp.info/en/mamp/) (Windows, Mac OS X)
```

**Review by matks — APPROVED** [review]

**Comment by okom3pom** [comment]
Hi Math i check the link in https://archive.org/web/ normaly it's work.

If you use windows fr : 

https://www.mamp.info/ 
Redirect to 
https://www.mamp.info/fr/
Redirect to 
https://www.mamp.info/fr/windows/

**Comment by matks** [comment]
> Hi Math i check the link in https://archive.org/web/ normaly it's work.
> 
> If you use windows fr :
> 
> https://www.mamp.info/
> Redirect to
> https://www.mamp.info/fr/
> Redirect to
> https://www.mamp.info/fr/windows/

🤔 Interesting

I'm on a Mac computer and https://www.mamp.info/ returns HTTP 403 to me 😮 

Maybe it's a temporary issue, let's wait a few hours to see if it is fixed

**Comment by okom3pom** [comment]
https://www.mamp.info/en/ only is good

**Comment by okom3pom** [comment]
Yes it's an 403 but i think it's a temporary issue

We can use https://www.mamp.info/en/

Haha i use ubuntu :)

Capture d’écran de 2020-10-04 15-58-33

**Comment by matks** [comment]
https://www.mamp.info/ still a 403 😢 for me (so possibly all Mac users)

But indeed https://www.mamp.info/en/ seems OK

**Comment by PierreRambaud** [comment]
FYI I'm on linux and https://www.mamp.info/  redirect me to https://www.mamp.info/en/windows/ :roll_eyes: 
Let's use https://www.mamp.info/en/ instead

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
> FYI I'm on linux and https://www.mamp.info/ redirect me to https://www.mamp.info/en/windows/ 🙄
> Let's use https://www.mamp.info/en/ instead

Very very funny for a project that used to run on Mac only 😆 maybe we should tell them

**Comment by PierreRambaud** [comment]
> > FYI I'm on linux and https://www.mamp.info/ redirect me to https://www.mamp.info/en/windows/ roll_eyes
> > Let's use https://www.mamp.info/en/ instead
> 
> Very very funny for a project that used to run on Mac only  maybe we should tell them

My secret has been discovered, everyone now knows that I'm working under Windows :scream:

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @okom3pom

**Pull request merged**